prov/gni/src/gnix_ep.c

```diff
@@ -429,18 +432,19 @@ int gnix_ep_open(struct fid_domain *domain, struct fi_info *info,
 	 */
 
 	if (nic) {
-		gnix_list_del_init(&nic->list);
-		list_add_tail(&gnix_nic_list, &nic->list);
+		gnix_list_del_init(&nic->gnix_nic_list);
+		list_add_tail(&gnix_nic_list, &nic->gnix_nic_list);
 	}
 
 	/*
 	 * no nic found create a cdm and attach
 	 */
 	if (!nic) {
```

Comment:
We could disable that check when we call the pre-commit script.  We already disable the CONTRIBUTORS warning.